app/views/layouts/application.html.erb

```diff
@@ -25,7 +25,10 @@
     <% end %>
 
     <div id="vue-app">
-      <the-header></the-header>
+      <the-header
+        :signed-in="<%= current_user.present? %>"
```

Comment:
@gmq con lo que preguntaron recién en slack me entró la duda. Le agregué una prop, entonces técnicamente ya no debería tener the, pero nunca va a haber más de un header

Reply:
Creo que la excepción es cuando usamos props que vienen rails, ya que al final no es algo que estemos modificando desde Vue. 🤔

Reply:
ah puede ser 🤔. Como que me causa duda lo categórico que son en la guía de estilo con decir que nunca reciben props, como que hay casos que me hace sentido que algo sea único en una vista y necesite props

Reply:
Me da la impresión que asumen que estás trabajando en una SPA asi que si necesitas información no llegaría de un parent, llegaría del state o algo así.

Reply:
ahh interesante, igual tiene sentido